www/newitems.php

```diff
@@ -425,7 +425,7 @@ function showNewItemList($db, $items, $first, $last, $showFlagged, $allowHiddenB
                     echo "<i>\"$summary\"</i>";
 
                 if ($trunc)
-                    echo " - <a href=\"viewgame?id={$r['gameid']}"
+                    echo " - <a class=eager href=\"viewgame?id={$r['gameid']}"
```

Comment:
mix of unquoted and quoted HTML attributes (harmless, similar issue in other lines)